build.gradle

```
@@ -0,0 +1,33 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.1.5'
```

```suggestion
	id 'org.springframework.boot' version '3.1.5'  apply 'false'
```

https://docs.gradle.org/current/dsl/org.gradle.plugin.use.PluginDependenciesSpec.html#org.gradle.plugin.use.PluginDependenciesSpec:id(java.lang.String)